src/test/lambda/wizards/samInitWizard.test.ts

```diff
@@ -37,6 +43,18 @@ class MockCreateNewSamAppWizardContext implements CreateNewSamAppWizardContext {
         return (this._lambdaRuntimes as Set<Runtime>) || Set()
     }
 
+    public get samTemplates(): Set<SamTemplate> {
+        if (Array.isArray(this._samTemplates)) {
+            if (this._samTemplates!.length <= 0) {
+                throw new Error('samTemplates was called more times than expected') // bunu belke rename ele
```

Comment:
Mystery comment

Reply:
:)) Sometimes I leave comments in my native language. Removed.